City-of-Helsinki/drupal-helfi-form-tool
# Feature/lom 274 seurantasivu

# [LOM-274](https://helsinkisolutionoffice.atlassian.net/browse/LOM-274) - Seurantasivu
Fixes the seurantasivu.

## What was done
* Seurantasivu is now as it should be. 

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/LOM-274-seurantasivu`
  * `make fresh`
* Run `make drush-cr`

## How to test
* Submit a form 
* When you have the option to "View the data you submitted" click the link
* See that the page works on desktop and mobile.

* [x] See that the page matches [the layout](https://share.goabstract.com/c5d70fb0-a4a0-4878-86dc-ac2aa9c3d551?collectionLayerId=5973ce9d-7adf-4301-9e0a-80541d1b1a84&mode=build&selected=root-2AFABB81-9EE1-48AE-8E28-B531A7D09DB5&sha=latest)
* [x] Check that this bug ticket is addressed: [LOM-303: Tarkista lähetetyn lomakkeen tiedot sivulla ylimääräistä tekstiä](https://helsinkisolutionoffice.atlassian.net/browse/LOM-303)
* [x] Check that this bug ticket is addressed: i [LOM-310: Lomakkeen tiedot sivulla teksti näkyy vain osittainOBSOLETE](https://helsinkisolutionoffice.atlassian.net/browse/LOM-310)

## Designers review


* [ ] This PR has been visually reviewed by a designer (Heikki)



[LOM-274]: https://helsinkisolutionoffice.atlassian.net/browse/LOM-274?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
